packages/react/src/Hint/Hint.test.tsx

```diff
@@ -0,0 +1,41 @@
+import { render } from '@testing-library/react'
+import { createRef } from 'react'
+import { Hint } from '.'
+import '@testing-library/jest-dom'
+
+describe('Hint', () => {
+  it('renders', () => {
+    const { container } = render(<Hint />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toBeInTheDocument()
+    expect(component).toBeVisible()
+  })
+
+  it('renders a design system BEM class name', () => {
+    const { container } = render(<Hint />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('ams-hint')
+  })
+
+  it('renders an additional class name', () => {
+    const { container } = render(<Hint className="extra" />)
+
+    const component = container.querySelector(':only-child')
+
+    expect(component).toHaveClass('ams-hint extra')
+  })
+
```

Comment:
We're missing a test for the `hint` prop

Reply:
Has been added, see below.